worker-batch/pom.xml

```diff
@@ -140,20 +141,15 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>javax.el</groupId>
-            <artifactId>javax.el-api</artifactId>
+            <groupId>jakarta.el</groupId>
+            <artifactId>jakarta.el-api</artifactId>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.hibernate.validator</groupId>
             <artifactId>hibernate-validator</artifactId>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>com.hpe.caf.worker.batch</groupId>
-            <artifactId>worker-batch-test-plugin</artifactId>
```

Comment:
Removed this dependency to avoid the conflict that's happening.

> [WARNING] Found duplicate and different classes in [com.github.jobservice:worker-batch-extensibility:3.7.0-SNAPSHOT, com.hpe.caf.worker.batch:worker-batch-extensibility:1.11.0-16]:
> [WARNING]   com.hpe.caf.worker.batch.BatchDefinitionException
> [WARNING]   com.hpe.caf.worker.batch.BatchWorkerPlugin
> [WARNING]   com.hpe.caf.worker.batch.BatchWorkerServices
> [WARNING] Found duplicate classes/resources in test classpath.